README.md

```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/healthchecks-thunknalla.svg?branch=master)](https://travis-ci.org/andela/healthchecks-thunknalla)
```

Comment:
Drop the query string that specifies the branch as `master`, for the badge to show the build status of the most recent build. i.e `https://travis-ci.org/andela/healthchecks-thunknalla.svg`